build/__rubyVersions.sh

```diff
@@ -7,3 +7,5 @@ RUBY26_VERSION='2.6.6'
 RUBY26_TAR_SHA256='5db187882b7ac34016cd48d7032e197f07e4968f406b0690e20193b9b424841f'
 RUBY25_VERSION='2.5.8'
 RUBY25_TAR_SHA256='0391b2ffad3133e274469f9953ebfd0c9f7c186238968cbdeeb0651aa02a4d6d'
+RUBY31_VERSION='3.1.1'
+RUBY31_TAR_SHA256='fe6e4782de97443978ddba8ba4be38d222aa24dc3e3f02a6a8e7701c0eeb619d'
```

Comment:
I could be possibly wrong, but while creating my PR I found a different SHA

RUBY31_TAR_SHA256='7aefaa6b78b076515d272ec59c4616707a54fc9f2391239737d5f10af7a16caa'